ci/import-squidex-data.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+
+# create-app.py configures sq to new app
+sq sync in backup 2>&1 | tee sq-sync-log.txt
```

Comment:
do you need to store it in a file? cant you simply store in some env var? surely you could still grep it?

Reply:
i started off with `if sq sync in backup | grep` but that hid all of the sync output from the pipeline, this seems to be the easiest way to get gitlab to keep piping the results to the pipeline (in realtime). a bit clunky but the logs are deleted at the end

Reply:
you could store it in an env variable 
```bash
export output="$(sq sync in backup 2>&1)"
```
also if sq sync in does not actually error during failures do you need to redirect stderr to stdout?

anyway once you have it in the env file i dont think we need to see the output unless it was a failure, in which case you can just echo it (as in - only when it fails).

I'd also add some other echo like 'Importing data...' before the command and 'Import successful' after and i think thats really all that matters. Alternatively on success you could have an env variable which is a flag that you add it it will also display the output on success (for debugging). Otherwise i think this output is just noise, i really think this whole process should be a separate job.

Reply:
have implemented all the above, with the exception of the separate job, was taking me a while to figure out the gitlab yaml stuff. have joined that ticket https://trello.com/c/8trOLvTW/1522-separate-squidex-import-from-the-sls-deploy-to-speed-up-the-pipeline

Reply:
oh yeah i didnt want you to split it now, of course